lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -596,7 +614,8 @@ checkpointFromEntity cp utxo s =
         ]
     blockHeight' = Quantity . toEnum . fromEnum $ bh
     bp = W.BlockchainParameters
-        { getGenesisBlockDate = W.StartTime genesisStart
+        { getGenesisBlockHash = coerce genesisHash
```

Comment:
At the moment all of the stuff in BlockchainParameters is effectively static because it comes from the genesis block, and there is no way to get parameter updates. Genesis block date, hash, and slot duration will never change.

The genesis hash is needed to instantiate the TransactionLayer.